pages/science/reactions.rst

```diff
@@ -154,6 +154,15 @@ three parameters, :math:`(A, T_3, T_1)`, are required. The fourth parameter,
 :math:`T_2`, is optional; if omitted, the last term of the falloff function is
 not used.
 
+Tsang's Approximation to :math:`F_{cent}`
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Wing Tsang presented approximations for the value of :math:`F_{cent}` for Troe falloff in databases of reactions, e.g. [#Tsang1991]_.
+Tsang's approximations are either constant or linear in temperature, i.e. :math:`F_{cent} = A + B*T`.
+The remaining equations for :math:`C`, :math:`N`, :math:`f_1`, and :math:`F` are not affected.
```

Comment:
Please avoid using Latin abbreviations (e.g., i.e.). They introduce yet one more barrier to entry for anyone reading this.

Reply:
Done